packages/calcite-components/src/components/flow-item/flow-item.tsx

```diff
@@ -69,6 +69,23 @@ export class FlowItem
   /** When `true`, the component will be hidden. */
   @Prop({ reflect: true }) closed = false;
 
+  /**
+   * When `true`, hides the component's content area.
```

Comment:
Nice, I like this description. +@geospatialem for an extra pair of eyes. ➕👀➕

Reply:
Awaiting @geospatialem's approval 👀 🙏

Reply:
![seal-approval](https://github.com/Esri/calcite-design-system/assets/5023024/3f58e4d4-4f60-45b4-b7eb-4ecd0bf7343f)